### DISCUSSION TITLE
make tailwindcss optional in CLI

### DISCUSSION THREAD

**Discussion by zhihengGet** [huntabyte/shadcn-svelte#197] [General]
amazing UI library ! 

i wouldl like to use it without tailwindcss but with unoCSS or other css frameworks, can we make tw optional in cli and make it just download the component without checking anything ? `npx shad-cn --ignore`

i can setup css variables required manually with unnoCSS, and it works ok,  and tailwind-variants actually works with any  css engine that understand tw classes not just tw....

p.s i think doc is missing bi

**Comment by huntabyte** [comment]
The entire project is built on top of TailwindCSS, so this isn't something on the roadmap at this time. You're more than welcome to copy the code over and make style adjustments in your CSS framework of choice though!
